### ISSUE TITLE
Chapter: Think exponential

### ISSUE THREAD

**Issue by lukefretwell** [proudlyserving/book#79]
M = Maintainers (@rebeccawoodbury @lukefretwell @marlena-medford)
C = Contributors (@CivicPossible)

Reference:

* [Chapter doc](https://docs.google.com/document/d/1KOfXKmGMY2x43JRjDOfFgmyHdNoU6OHXAxQ3edv42Uc/edit?usp=sharing)
* Questions? Ask in chapter doc using comments feature or via [contact form](https://proudlyservingbook.com/contact)

### Set-up

- [x] Review contributor request (M - @rebeccawoodbury, @lukefretwell @marlena-medford)
- [x] Create a [new issue](https://github.com/proudlyserving/book/issues/new?assignees=lukefretwell%2C+rebeccawoodbury%2C+Marlena-Medford&labels=content&template=new-chapter.md&title=Chapter%3A+%5BTITLE%5D) (M - @lukefretwell)
- [x] Send contributors 'Welcome contributors' email (M - @rebeccawoodbury)
- [x] Schedule kick-off meeting (C)
- [x] Add @rebeccawoodbury @lukefretwell @Marlena-Medford to meeting invite (M - @rebeccawoodbury)
- [x] Create new chapter doc from [chapter template](https://docs.google.com/document/d/16oS3naY8zCbcjr2QoJ6JvOk8YeJPsQ0ZENFec8gMshQ/edit?usp=sharing) (M - @rebeccawoodbury)
- [x] In file title and doc h1, replace 'TITLE' with the title of the chapter (example chapter) (M - @rebeccawoodbury)
- [x] Add link to GitHub issue in chapter doc (@rebeccawoodbury)
- [x] Meet for kick-off (C / M - @rebeccawoodbury @lukefretwell @marlena-medford)
- [x] Share edit rights with contributors and maintainers (M - @rebeccawoodbury)
- [x] Set two-week [calendar deadline reminder for writing chapter draft (+ add contributors in [Proudly Serving calendar](https://calendar.google.com/calendar/u/3?cid=YXRqdmRlY3MxMHU4ZmI2N2ZwbGNsazh0NzRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) / [template text](https://docs.google.com/document/d/1JzJrG2J7WzbtGK-A3TgSVCkyXHJJFuqriVn_vIxTj-8/edit#heading=h.wxmd11mqg67) (M - @rebeccawoodbury)
- [x] Set three-week calendar deadline reminder for maintainer review (+ add contributors in [Proudly Serving calendar](https://calendar.google.com/calendar/u/3?cid=YXRqdmRlY3MxMHU4ZmI2N2ZwbGNsazh0NzRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) / [template text](https://docs.google.com/document/d/1JzJrG2J7WzbtGK-A3TgSVCkyXHJJFuqriVn_vIxTj-8/edit#heading=h.wxmd11mqg67)) (M - @rebeccawoodbury)
- [x] Set five-week calendar deadline reminder for public feedback (+ add contributors in [Proudly Serving calendar](https://calendar.google.com/calendar/u/3?cid=YXRqdmRlY3MxMHU4ZmI2N2ZwbGNsazh0NzRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) / [template text](https://docs.google.com/document/d/1JzJrG2J7WzbtGK-A3TgSVCkyXHJJFuqriVn_vIxTj-8/edit#heading=h.wxmd11mqg67)) (M - @rebeccawoodbury)
- [x] Set six-week calendar deadline reminder to finalize chapter (+ add contributors in [Proudly Serving calendar](https://calendar.google.com/calendar/u/3?cid=YXRqdmRlY3MxMHU4ZmI2N2ZwbGNsazh0NzRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) / [template text](https://docs.google.com/document/d/1JzJrG2J7WzbtGK-A3TgSVCkyXHJJFuqriVn_vIxTj-8/edit#heading=h.wxmd11mqg67)) (M - @rebeccawoodbury)
- [x] Add contributors to [GitHub contributors team](https://github.com/orgs/proudlyserving/teams/proudly-serving-contributors) (M - @lukefretwell)
- [x] Add contributors to this GitHub issue (M - @lukefretwell)
- [x] Add doc link to this issue 'Reference' above (M - @lukefretwell)
- [x] Review [style guide](https://proudlyservingbook.com/style) (C - @CivicPossible)
- [x] Review [code of conduct](https://proudlyservingbook.com/conduct) (C- @CivicPossible)
- [x] Add headshots to GitHub issue (C- @CivicPossible)
- [x] Add contributor(s) to contributor list ([doc](https://docs.google.com/document/d/1rruJsEF8-E3qTVCv0Giw2mK43HcNS4d7233rgGk9wjw/edit?usp=sharing)/[GitBook](https://proudly-serving.gitbook.io/proudly-serving/)/[website](https://proudlyservingbook.com/people/) (M - @lukefretwell)
- [x] Add 'Proudly Serving contributor' to LinkedIn profile (C- @CivicPossible)
- [x] Send contributors 'Kick-off follow-up' email (M - @rebeccawoodbury)

### Write draft

- [x] Add bios/social media links to ‘Contributors’ section (C- @CivicPossible)
- [x] Complete first draft (2 weeks after kick-off)  (C- @CivicPossible)
- [x] When ready for review, highlight doc h1, add a ‘Ready for review’ comment and assign it to proudlyservingbook@gmail.com (C- @CivicPossible)
- [x] #85
- [x] Select [icon](https://thenounproject.com/), add to [colophon](https://proudlyservingbook.com/colophon) (M - @lukefretwell)
- [x] Notify contributors to review/approve final edits (M - @rebeccawoodbury)
- [x] When ready for public feedback, highlight doc h1, add a ‘Ready for feedback' comment and assign it to proudlyservingbook@gmail.com (C- @CivicPossible)

### Promote feedback

* [Content templates](https://docs.google.com/document/d/1JzJrG2J7WzbtGK-A3TgSVCkyXHJJFuqriVn_vIxTj-8/edit#heading=h.1yfb0xjocjrm)

- [x] [Blog](https://proudlyservingbook.com/updates/) (M - @rebeccawoodbury)
- [x] [Substack](https://proudlyserving.substack.com/) (M - @rebeccawoodbury)
- [x] [Twitter](https://twitter.com/proudly_serving) (M - @marlena-medford)
- [x] [LinkedIn](https://www.linkedin.com/company/proudlyserving) (M - @marlena-medford)
- [x] Send email to Contributor asking them to help promote on social (M - @rebeccawoodbury)
- [x] Share on social (C- @CivicPossible)

### Finalize feedback

- [x] Respond to all comments and incorporate where appropriate (C- @CivicPossible)
- [x] When ready for final publish, highlight doc h1, add a ‘Ready to publish' comment and assign it to proudlyservingbook@gmail.com (C- @CivicPossible)
- [x] Review for copy edits ( M - @marlena-medford)
- [x] Deprecate doc (set to view only, add ‘This document has been deprecated. Visit the Proudly Serving website (https://proudlyservingbook.com/) to learn how to give feedback on this chapter.‘) (M - @lukefretwell)

### Publish

- [x] [Doc](https://docs.google.com/document/d/1rruJsEF8-E3qTVCv0Giw2mK43HcNS4d7233rgGk9wjw/edit?usp=sharing) (M - @lukefretwell)
- [x] [GitBook](https://proudly-serving.gitbook.io/proudly-serving/) (M - @lukefretwell)

### Promote published

* [Content templates](https://docs.google.com/document/d/1JzJrG2J7WzbtGK-A3TgSVCkyXHJJFuqriVn_vIxTj-8/edit#heading=h.1yfb0xjocjrm)

- [x] [Blog](https://proudlyservingbook.com/updates/) (M - @rebeccawoodbury)
- [x] [Substack](https://proudlyserving.substack.com/) (M - @rebeccawoodbury)
- [x] [Twitter](https://twitter.com/proudly_serving) (M - @marlena-medford)
- [x] [LinkedIn](https://www.linkedin.com/company/proudlyserving) (M - @marlena-medford)
- [x] Send email to Contributor asking them to help promote on social (M - @rebeccawoodbury)
- [x] Share on social (C- @CivicPossible)
- [x] Send contributors 'Thank you' email (M - @rebeccawoodbury)

## Reference

```
When we talk about digital government services, there’s often a particular aspect that’s emphasized, such as the technology, procurement process or delivery model. Rarely do we hear a holistic vision for how to execute from an exponential perspective and bring digital agility to scale for civic-focused institutions.
While we’re beginning to see the emergence of digital services teams, the government technology community is still largely executing technology at a more bespoke, expensive approach to service delivery. This works for large organizations, such as federal and (some) state agencies and large cities, however, as the organization, and its resources, get smaller, that bespoke, expensive approach isn’t an option.
These smaller organizations must think exponentially when it comes to digital government.
Exponential digital government
I’ve been working on my thoughts around ‘Exponential Digital Government‘ and have started to outline principles and a general technology approach to help explain a vision for how this works.
Principles
Before taking a technology-first prescription to digital government services, I wanted to create some fundamental operating principles that serve as a foundation to help drive technology decision-making. Here’s where I am currently.
Government technology should be:
Sustainable
Scalable
Open
Evolving
Empowered
Technology
From a technology perspective, I wanted to think about how best to execute on these principles. Here’s where I am currently.
Continuously improving, cost-effective government technology is:
Platform-based
Open source
Extensible
Data-enabled
Software-as-a-service (+ OpenSaaS)
I’m continuing to refine this concept of exponential digital government and would love feedback from the community. Please feel free to add your comments to the deck. In the near future, I will publish a more refined, ‘final’ version.
Regardless of whether my initial draft is the perfect framework to adopt, remind yourself to think exponential when it comes to digital government services.



```

**Comment by rebeccawoodbury** [comment]
@lukefretwell bio and headshot for Jason: 

Jason is an altruistic mischief maker with [Civic* Possible](https://www.civicpossible.com/) helping community leaders solve sticky, cross silo challenges.  He has 20 years’ experience playing in the intersection of government, nonprofits, and business. He has served as an elected City Commissioner, Alaska’s first Innovation Officer, executive director of NGOs, start up business coach, and the Economic Development lead for Marion County Oregon. 

![Jason-headshot](https://user-images.githubusercontent.com/19391974/139482748-3c0f0366-8d2b-4340-8146-685e0a8af390.jpg)

**Comment by lukefretwell** [comment]
@rebeccawoodbury @civicpossible new page! https://proudlyservingbook.com/people/

**Comment by rebeccawoodbury** [comment]
@lukefretwell we need to add a bullet for "add chapter to feedback page on website" to our workflow when we promote it for public feedback

**Comment by lukefretwell** [comment]
@rebeccawoodbury done. I added it to the template, but we'll need to insert into any of the old ones.

**Comment by lukefretwell** [comment]
@rebeccawoodbury ready to blog/newsletter: https://proudlyservingbook.com/think-exponential cc @Marlena-Medford

**Comment by rebeccawoodbury** [comment]
@lukefretwell we need to reorder the Promote published section. We need social links before we can publish post and substack.

**Comment by rebeccawoodbury** [comment]
@lukefretwell OMG I just realized I could drag and drop the order of the checklist items. So I just did.

**Comment by lukefretwell** [comment]
@rebeccawoodbury I updated the order in the template, but the other opened ones will need to be updated.
